src/ValidatorSchema.php

```diff
@@ -52,12 +56,16 @@ private function checkInfo(): void
      */
     private function checkServers(): void
     {
-        if ($this->definition['servers']) {
+        // check exists section servers.
+        if (!array_key_exists('servers', $this->definition)) {
             return;
         }
 
-        if (!isset($this->definition['servers']['url'])) {
-            throw DefinitionCheckerException::missingField('servers.url');
+        // check required props of server.
+        foreach ($this->definition['servers'] as $server) {
+            if (empty($server->jsonSerialize()['url'])) {
+                throw DefinitionCheckerException::missingField('servers.url');
+            }
         }
     }
```

Comment:
I was removing this method because url isn't a field like info and is mandatory when instantiating a Server object. Do you mind removing it ?